vispy/testing/_runners.py

```diff
@@ -190,8 +190,15 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(fnames_str):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
```

Comment:
haha, okay.
I'm wondering why sublime text's ```flake8``` plugin doesn't lint docstrings too. Or is that not part of the same set of standards?